addons/common/fnc_isTurnedOut.sqf

```diff
@@ -27,54 +27,53 @@ Examples:
 
 Author:
     Nou, courtesy of ACRE project.
+    shukari
 ---------------------------------------------------------------------------- */
 SCRIPT(isTurnedOut);
 
 params ["_unit"];
 
-private _turn = false;
-private _out = false;
+if (isNull objectParent _unit) exitWith {true};
+if (isTurnedOut _unit) exitWith {true};
+
+private _return = false;
 private _vehicle = vehicle _unit;
-private _cfg = configFile >> "CfgVehicles" >> (typeOf _vehicle);
-
-if (_vehicle != _unit) then {
-    if (driver _vehicle == _unit) then {
-        private _forceHideDriver = getNumber(_cfg >> "forceHideDriver");
-        _turn = (_forceHideDriver != 1);
-        _out = (_vehicle animationPhase "hatchDriver") > 0;
-    } else {
-        private _assignedRole = assignedVehicleRole _unit;
-        if (_assignedRole select 0 == "Turret") then {
-            private _turretPath = _assignedRole select 1;
-            private _turret = [_vehicle, _turretPath] call CBA_fnc_getTurret;
-
-            private _canHideGunner = getNumber(_turret >> "canHideGunner");
-            private _forceHideGunner = getNumber(_turret >> "forceHideGunner");
-            _turn = (_canHideGunner == 1 && _forceHideGunner == 0);
-
-            private _hatchAnimation = getText(_turret >> "animationSourceHatch");
-            _out = (_vehicle animationPhase _hatchAnimation) > 0;
-
-        } else {
-            if ((_assignedRole select 0) == "Cargo") then {
-                private _attenuateCargo = getArray(_cfg >> "soundAttenuationCargo");
-                if ((count _attenuateCargo) > 0) then {
-                    private _index = (count _attenuateCargo)-1; // wait for command to get cargo index
-
-                    if (_index > -1) then {
-                        if(_index > (count _attenuateCargo)-1) then {
-                            _index = (count _attenuateCargo)-1;
-                        };
-                        if((_attenuateCargo select _index) == 0) then {
-                            _out = true;
-                        };
-                    };
-                };
-            };
-        };
+private _role = _unit call CBA_fnc_vehicleRole;
+
+// FFVs are allways outside
+if ([_unit] call ace_common_fnc_getTurretIndex in ([_vehicle] call ace_common_fnc_getTurretsFFV)) then {_return = true};
```

Comment:
ACE functions in CBA?

Reply:
AHAHAHAHAHAHAHAHAHAHAHAHAHA 😱 
There is the attention -> https://github.com/CBATeam/CBA_A3/pull/1015#issue-224558602
There is the first comment about the question -> https://github.com/CBATeam/CBA_A3/pull/1015#issuecomment-431695148
There is dedman talking about that -> https://github.com/CBATeam/CBA_A3/pull/1015#issuecomment-431797475

Reply:
Either github still have issues or I just overlooked those comments. And attention was overlooked for sure :)

Reply:
you are right... github is partially down atm https://status.github.com/messages

Reply:
>There is dedman talking about that

That's not my name

Reply:
Neither is dedmen, it's just your nickname. ;)